File: go/wam/2474.go
const WamUserNoticeError = 2474 // channel: regular

type UserNoticeErrorEvent struct {
	UserNoticeId int64 `wam:"user_notice_id"` // field 1
	UserNoticeContentVersion int64 `wam:"user_notice_content_version"` // field 2
	UserNoticeErrorEvent USERNOTICEERROREVENT `wam:"user_notice_error_event"` // field 3
	NoticeType NOTICETYPE `wam:"notice_type"` // field 4
}

File: rust/wam/2474.rs
pub const WAM_USER_NOTICE_ERROR: u32 = 2474;

#[derive(Debug, Default)]
pub struct UserNoticeErrorEvent {
    /// field 1, wire `user_notice_id`
    pub user_notice_id: Option<i64>,
    /// field 2, wire `user_notice_content_version`
    pub user_notice_content_version: Option<i64>,
    /// field 3, wire `user_notice_error_event`
    pub user_notice_error_event: Option<USERNOTICEERROREVENT>,
    /// field 4, wire `notice_type`
    pub notice_type: Option<NOTICETYPE>,
}